source/leuven/data_management_guidelines.rst

```diff
@@ -1,189 +1,147 @@
-multiple directories, making it easier to collaborate and find data of past projects. Switching the user directory and project directory would only make sense if there is only a single user per
-project, and in that case you are probably better of using ``$VSC_SCRATCH``.
+Structuring your files and directories
+======================================
 
-Divide input, intermediate and output data directories
-------------------------------------------------------
+It is worth spending some time on a suitable directory structure and file naming convention for your project, especially in the case of shared storage. We recommend to agree on a clear approach within your team (e.g. definining in advance which directory structure to use). This helps to avoid loosing data in scattered directories which multiple users would otherwise tend to create. This is best clearly documented so that any new collaborators can also follow these guidelines.
 
-While you are probably already making this division on a certain level in your directory tree, this paragraph is more focused on deciding on what level you should be making it. This will mostly depend
-on the level of data sharing within your team, and on the potential need of syncing your input data to your safe storage.
+Many best practices are already described on the `RDM file organization <https://www.kuleuven.be/rdm/en/guidance/data-standards/file-organisation>`_ page. One of the main points is to have a logical hierarchy with higher level broader topics and more specific folders within. A simple structure that tends to work well is to have a top-level directory for the research project which contains a subdirectory for each collaborator (possibly based on VSC IDs). Individual experiments can then be placed in separate directories within the user's subdirectory. If needed, related projects can be placed in a common parent directory.
 
-If you are sharing input data (or perhaps certain enrichment databases) with other teammembers, it would make sense to place these in a separate directory on a higher level than the experiment directory
-you need that input data for. How much higher will depend on the level of sharing. If you have certain data that is needed over different projects, it is clear that you could place this in a separate input
-directory above the project level. You could then opt for a top-level ``input_data`` and ``output_data`` directory, potentially with mirrored project directories in both (in case you have project-specific
-input data as well).
+As mentioned before, also think about where (shared) data and software are best located. Input datasets, enriched databases and software may be specific to one experiment or be used across different projects (and everything in between) and their position in the hierarchy should reflect this.
```

Comment:
'Input datasets, enriched databases and software may be specific to one experiment or be used across different projects (and everything in between) and their position in the hierarchy should reflect this.'

Should we include some more concrete example structures here? This is does contain all info, but I'm not really sure if it will be clear what we mean with this